tests/lib/snaps/serial-port-hotplug/bin/consumer

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+check_access() {
+    DEVPATH=$1
+    if test -w "$DEVPATH" ; then
```

Comment:
Is this sufficient to check if the device is really writable? I think it will only show it is seemingly writable. The device cgroup will still prevent access at the last stop. Test is probably implemented with `access` and that only looks at the file mode.

Reply:
I've changed it to really write data to the simulated port (and changed qemu to use regular file for the serial ports devices).